test/liblll/EndToEndTest.cpp

```diff
@@ -165,6 +165,31 @@ BOOST_AUTO_TEST_CASE(conditional_seq)
 	BOOST_CHECK(callFallback() == toBigEndian(u256(1)));
 }
 
+BOOST_AUTO_TEST_CASE(conditional_nested)
```

Comment:
This test could be merged nevertheless. Perhaps another one too with the a waterfall in the else clause.